[DOCS-12615] Simplify next js RUM guide (#33662)
DataDog/documentation: content/en/real_user_monitoring/guide/monitor-your-nextjs-app-with-rum.md

## Overview

[Next.js][1] is a JavaScript framework created by [Vercel][7] for building React web applications and Node.js APIs. This guide shows you how to add Datadog RUM to your Next.js application to monitor frontend performance and user behavior.

## Prerequisites

Create a RUM application in Datadog to get your credentials:

1. Go to [**Digital Experience > Performance Summary**][2].
2. Click **New Application**.

> 3. Select **JS**, enter an application name, and click **Create New RUM Application**.

Reviewer: To confirm, this is what I see, but ignore this suggestion if it's wrong.

```suggestion
3. Select **JS**, enter an application name, and click **Create Application**.
```

4. Copy the `applicationId` and `clientToken` values.

<div class="alert alert-info">If you store these in <code>.env.local</code>, prefix them with <code>NEXT_PUBLIC_</code> to expose them to the browser. See <a href="https://nextjs.org/docs/pages/building-your-application/configuring/environment-variables#bundling-environment-variables-for-the-browser" target="_blank">Next.js environment variables</a>.</div>

## Install the RUM SDK

> Choose the section below that matches your Next.js routing setup (App Router or Page Router).

Reviewer: ```suggestion
See the following sections for more information on the routing setup that matches your Next.js configuration (App Router or Page Router).
```

        `}
      </Script>
      <Component {...pageProps} />
    </>
  );
}
{{< /code-block >}}

{{% /tab %}}
{{% tab "CDN sync" %}}

Use `strategy="beforeInteractive"` to load RUM synchronously before the page becomes interactive:

{{< code-block lang="tsx" filename="pages/_app.tsx" disable_copy="false" >}}
import type { AppProps } from "next/app";
import Script from "next/script";

export default function App({ Component, pageProps }: AppProps) {
  return (
    <>
      <Script
        id="dd-rum-sync"
        src="https://www.datadoghq-browser-agent.com/us1/v6/datadog-rum.js"
        type="text/javascript"
        strategy="beforeInteractive"
      />
      <Script id="datadog-rum">
        {`
          window.DD_RUM && window.DD_RUM.init({
            clientToken: '<CLIENT_TOKEN>',
            applicationId: '<YOUR_APPLICATION_ID>',
            site: 'datadoghq.com',
            service: '<SERVICE_NAME>',
            env: '<ENV_NAME>',
            sessionSampleRate: 100,
            sessionReplaySampleRate: 20,
          });
        `}
      </Script>
      <Component {...pageProps} />
    </>
  );
}
{{< /code-block >}}

{{% /tab %}}
{{< /tabs >}}

## Verify and deploy

> 1. Run your app locally and check the browser console for RUM initialization.

Reviewer 2: I was reading what we do on other places like [this](https://docs.datadoghq.com/real_user_monitoring/application_monitoring/browser/setup/client/?tab=npm#:~:text=Step%204%20%2D%20Visualize,starts%20receiving%20data.)

What do you think about something like this?

## Verify and deploy
1. Run your app locally and visualize the [data collected](https://docs.datadoghq.com/real_user_monitoring/application_monitoring/browser/data_collected/) in [dashboards](https://docs.datadoghq.com/real_user_monitoring/platform/dashboards/) or create a search query in the [RUM Explorer](https://app.datadoghq.com/rum/sessions).
2. Deploy your changes. (Don't forget to change your env and version so you can find your production sessions better)

> 2. Deploy your changes. After deployment, view your [collected data][3] in [RUM dashboards][4].

Author: ```suggestion
2. Deploy your changes. While doing so, don't forget to change your env and version so you can find your production sessions later.
```
